languages/rust/exercises/concept/options/tests/options.rs

```diff
@@ -0,0 +1,89 @@
+use options::*;
+
+#[test]
+fn test_reviving_dead_player() {
+    let level: u32 = 34;
+    let dead_player = Player {
+        health: 0,
+        mana: Some(0),
+        level,
+    };
+    let revived_player = dead_player.revive();
+    assert_eq!(revived_player.is_some(), true);
+
+    let revived_player = revived_player.unwrap();
+    assert_eq!(revived_player.health, 100);
+    assert_eq!(revived_player.mana, Some(100));
+    assert_eq!(revived_player.level, level);
+}
+
+#[test]
+#[ignore]
+fn test_reviving_alive_player() {
+    let level: u32 = 8;
+    let alive_player = Player {
+        health: 1,
+        mana: None,
+        level,
```

Comment:
```suggestion
        8,
```

Again, no need to break out a `level` variable.